src/index.js

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
 import App from './App';
+import rootReducer from './redux/configureStore';
 import 'bootstrap/dist/css/bootstrap.min.css';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const store = createStore(rootReducer);
```

Comment:
- [ ] The purpose of the `configureStore.js` file in the redux directory is to hold all logic for creating a store and combining the other reducers. 
Because of this reason, I will suggest you place this line in the `configureStore.js` file and let the `index.js` file deals with wrapping the main `App` component with the redux provider.